### PR TITLE
Add User-Agent header

### PR DESCRIPTION
**Pull request by sepetrov**
**Issue Ref**:  #38 
 
**Description**: 

Adds `User-Agent` header with format `kubeless-kafka-trigger-controller/<version>`.

[PR Description]

**TODOs**:
 - [x] Ready to review
 - [ ] Automated Tests
 - [ ] Docs

### CODE REVIEW

File: pkg/utils/event_sender.go [hunk 1]
```diff
@@ -51,6 +53,7 @@ func GetHTTPReq(funcName string, funcPort int, kafkaTopic, namespace, eventNames
 	if err != nil {
 		return nil, fmt.Errorf("Unable to create request %v", err)
 	}
+	req.Header.Set("User-Agent", fmt.Sprintf("kubeless/kafka-trigger-controller %s", version.Version))
```
Comment (andresmgot) [inline pkg/utils/event_sender.go]: I would set this as `kubeless-kafka-trigger-controller/%s` (typically this field is ID/version)

**Review by andresmgot — APPROVED** [review]
LGTM but I would change the header format a bit. WDYT?

File: pkg/utils/event_sender.go (hunk 1 above)
Comment (sepetrov) [reply pkg/utils/event_sender.go]: Updated

**Comment by andresmgot** [comment]
Thanks!

**Pull request merged**